prisma/schema.prisma

```diff
@@ -48,6 +48,7 @@ model Post {
   bookmarks     Bookmark[]
   comments      Comment[]
   notifications Notification[]
+  allowComments Boolean        @default(true)
```

Comment:
Very pedantic! Would you mind renaming it to `showComments`.

Thanks for this! 🙌

If you don't have time let me know and I can merge and make the change myself.